request_query_count/query_count.py

```diff
@@ -0,0 +1,267 @@
+import json
+import re
+import sys
+from sys import maxsize, stderr
+
+ANY = '<any>'
+
+
+class QueryCountExclusion(object):
```

Comment:
I'm using flake8 (plus `cop.sh`),  and it complains if you don't put exactly 2 lines between first level definitions. In this case `ANY = ...` and class `QueryCountExclusion...`, so I think this is right